### PR TITLE
Add pipeline for ras-rm-metrics

### PR DESCRIPTION
**Pull request by PurpleBooth**
# Motivation and Context

Automate the deployment of ONSdigital/ras-rm-metrics

# What has changed

This deploys ONSdigital/ras-rm-metrics on every environment.

# How to test?

Includes smoke tests.

* Visit https://ras-rm-metrics-latest.apps.devtest.onsclofo.uk/info
* Visit https://ras-rm-metrics-preprod.apps.devtest.onsclofo.uk/info
* Visit https://ras-rm-metrics-preprod.apps.devtest.onsclofo.uk/info

# Links

Depends on https://github.com/ONSdigital/ras-rm-metrics/pull/1

### CODE REVIEW

**Comment by PurpleBooth** [comment]
[Add RabbitMQ metrics into Splunk (1d)](https://trello.com/c/OONR0i8R/79-add-rabbitmq-metrics-into-splunk-1d)

**Comment by PurpleBooth** [comment]
[Add RabbitMQ metrics into Splunk (1d)](https://trello.com/c/OONR0i8R/79-add-rabbitmq-metrics-into-splunk-1d)

**Review by rjweeks — APPROVED** [review]

File: pipelines/ras-rm-metrics.yml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  type: docker-image
+  source:
+    repository: cfcommunity/slack-notification-resource
+
+resources:
+- name: ras-rm-metrics-source
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/ras-rm-metrics.git
+    branch: deploy_app
```
Comment (AdamHawtin) [inline pipelines/ras-rm-metrics.yml]: This should be pointing at master branch before merge

Comment (rjweeks) [reply pipelines/ras-rm-metrics.yml]: @AdamHawtin I've changed it to master for merge.

File: pipelines/ras-rm-metrics.yml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      environment_variables:
+        SCHEDULER_FREQUENCY: '15'
+        RABBITMQ_SERVICE_NAME: rabbitmq
+
+  - put: map-route-vpn-domain
+    resource: cf-cli-resource-preprod
+    params:
+      command: map-route
+      app_name: ras-rm-metrics-preprod
+      domain: rhpreprod.rmdev.onsdigital.uk
```
Comment (AdamHawtin) [inline pipelines/ras-rm-metrics.yml]: This doesn't look right

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: pipelines/ras-rm-metrics.yml (hunk 2 above)
Comment (rjweeks) [reply pipelines/ras-rm-metrics.yml]: Deleted this resource.

File: pipelines/ras-rm-metrics.yml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+      environment_variables:
+        SCHEDULER_FREQUENCY: '15'
+        RABBITMQ_SERVICE_NAME: rabbitmq
+
+  - task: smoke-tests
+    on_failure:
+      put: notify
+      params:
+        text:  |
+          Latest smoke tests failed. Check the build:
```
Comment (AdamHawtin) [inline pipelines/ras-rm-metrics.yml]: The smoke tests failure slack messages could do with having the service name in the message

Comment (rjweeks) [reply pipelines/ras-rm-metrics.yml]: Added.

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**